runtime/src/bpf_map/array_map_kernel_user.cpp

```diff
@@ -34,7 +34,7 @@ void array_map_kernel_user_impl::init_map_fd()
 	}
 	bpf_map_info info = {};
 	unsigned int info_len = sizeof(info);
-	int res = bpf_map_get_info_by_fd(map_fd, &info, &info_len);
```

Comment:
Why?

Reply:
I chose a version of libbpf that does not provide functions like bpf_{prog,map}_get_info_by_fd, use bpf_obj_get_info_by_fd, they are the same things